### DISCUSSION TITLE
Re-deploy, how can I handle it better?

### DISCUSSION THREAD

**Discussion by cvhainb** [webpack/webpack#14122] [Q&A] [answered]
Hello,

I use Laravel Mix to deploy my app then for some reason, I want to change something for my vue components and I must re-run "npm run production" again.

When use this command, my CPU is 100% and it takes about a few minutes. My questions is: How can I re-deploy my app better?

Thanks in advanced,

Chu

**Comment by alexander-akait** [comment, marked as answer]
Please ask this in `Laravel Mix` repo
